<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>508</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>943</width>
    <height>508</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>943</width>
    <height>508</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>8</y>
      <width>925</width>
      <height>265</height>
     </rect>
    </property>
    <property name="title">
     <string>Processing parameters</string>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>198</y>
       <width>907</width>
       <height>59</height>
      </rect>
     </property>
     <property name="title">
      <string>Components</string>
     </property>
     <widget class="QCheckBox" name="cbXComponent">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>26</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>x-component</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbYComponent">
      <property name="geometry">
       <rect>
        <x>126</x>
        <y>26</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>y-component</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbZComponent">
      <property name="geometry">
       <rect>
        <x>244</x>
        <y>26</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>z-component</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>78</y>
       <width>909</width>
       <height>63</height>
      </rect>
     </property>
     <property name="title">
      <string>Cumulative spectrum parameters</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>32</y>
        <width>95</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Window size</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>202</x>
        <y>32</y>
        <width>95</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Overlap size</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>638</x>
        <y>36</y>
        <width>95</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Median filter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>406</x>
        <y>34</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Marmett filter</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbWindowSize">
      <property name="geometry">
       <rect>
        <x>96</x>
        <y>28</y>
        <width>103</width>
        <height>26</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100000</number>
      </property>
      <property name="value">
       <number>8192</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbOverlapSize">
      <property name="geometry">
       <rect>
        <x>288</x>
        <y>28</y>
        <width>83</width>
        <height>26</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100000</number>
      </property>
      <property name="value">
       <number>4093</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbMarmettFilter">
      <property name="geometry">
       <rect>
        <x>508</x>
        <y>30</y>
        <width>63</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>999</number>
      </property>
      <property name="singleStep">
       <number>2</number>
      </property>
      <property name="value">
       <number>7</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbMedianFilter">
      <property name="geometry">
       <rect>
        <x>734</x>
        <y>30</y>
        <width>59</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>999</number>
      </property>
      <property name="singleStep">
       <number>2</number>
      </property>
      <property name="value">
       <number>7</number>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbUseMarmettFilter">
      <property name="geometry">
       <rect>
        <x>574</x>
        <y>32</y>
        <width>47</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>use</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbUseMedianFilter">
      <property name="geometry">
       <rect>
        <x>796</x>
        <y>32</y>
        <width>47</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>use</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>142</y>
       <width>425</width>
       <height>57</height>
      </rect>
     </property>
     <property name="title">
      <string>Correlation parameters</string>
     </property>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>32</y>
        <width>129</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Minimal frequency</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="sbMinCorrelationFrequency">
      <property name="geometry">
       <rect>
        <x>134</x>
        <y>28</y>
        <width>69</width>
        <height>26</height>
       </rect>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="sbMaxCorrelationFrequency">
      <property name="geometry">
       <rect>
        <x>348</x>
        <y>28</y>
        <width>69</width>
        <height>26</height>
       </rect>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
      <property name="value">
       <double>10.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>216</x>
        <y>32</y>
        <width>129</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Maximal frequency</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>432</x>
       <y>142</y>
       <width>425</width>
       <height>57</height>
      </rect>
     </property>
     <property name="title">
      <string>Visualization parameters</string>
     </property>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>32</y>
        <width>129</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Minimal frequency</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="sbMinVisualFrequency">
      <property name="geometry">
       <rect>
        <x>134</x>
        <y>28</y>
        <width>69</width>
        <height>26</height>
       </rect>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="sbMaxVisualFrequency">
      <property name="geometry">
       <rect>
        <x>348</x>
        <y>28</y>
        <width>69</width>
        <height>26</height>
       </rect>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
      <property name="value">
       <double>10.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>216</x>
        <y>32</y>
        <width>129</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Maximal frequency</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>911</width>
       <height>57</height>
      </rect>
     </property>
     <property name="title">
      <string>Signal Preprocessing</string>
     </property>
     <widget class="QDateTimeEdit" name="dtStopTime">
      <property name="geometry">
       <rect>
        <x>536</x>
        <y>26</y>
        <width>159</width>
        <height>26</height>
       </rect>
      </property>
      <property name="displayFormat">
       <string>dd.MM.yyyy HH:mm:ss</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbResampleFrequency">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>24</y>
        <width>77</width>
        <height>26</height>
       </rect>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>30</y>
        <width>73</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Start time</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>466</x>
        <y>30</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop time</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>28</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Resample frequency</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="dtStartTime">
      <property name="geometry">
       <rect>
        <x>302</x>
        <y>26</y>
        <width>161</width>
        <height>26</height>
       </rect>
      </property>
      <property name="displayFormat">
       <string>dd.MM.yyyy HH:mm:ss</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>698</x>
        <y>30</y>
        <width>135</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Detrend frequency</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="sbDentendEdge">
      <property name="geometry">
       <rect>
        <x>832</x>
        <y>26</y>
        <width>71</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>282</y>
      <width>927</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Output data</string>
    </property>
    <widget class="QCheckBox" name="cbSelectionSignalFile">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>52</y>
       <width>155</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Selection signal file</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbSelectionSignalGraph">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>78</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Selection signal graph</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbSpectrumGraph">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>104</y>
       <width>153</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Spectrum graph</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbUnSmoothSpectrumFile">
     <property name="geometry">
      <rect>
       <x>216</x>
       <y>54</y>
       <width>245</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Unsmooth spectrum file (*.sc)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbSmoothSpectrumFile">
     <property name="geometry">
      <rect>
       <x>216</x>
       <y>80</y>
       <width>229</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Smooth spectrum file (*.ssc)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbCorrelationMatrixFile">
     <property name="geometry">
      <rect>
       <x>508</x>
       <y>78</y>
       <width>243</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Correlation matrix file</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbCorrelationGraph">
     <property name="geometry">
      <rect>
       <x>508</x>
       <y>52</y>
       <width>229</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Correlation graph</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbGeneralCorrelationGraph">
     <property name="geometry">
      <rect>
       <x>508</x>
       <y>104</y>
       <width>245</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>General correlation graph</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbGeneralSmoothSpectrumGraph">
     <property name="geometry">
      <rect>
       <x>216</x>
       <y>104</y>
       <width>243</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>General smooth spectrum graph</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leFolderPath">
     <property name="geometry">
      <rect>
       <x>118</x>
       <y>24</y>
       <width>761</width>
       <height>25</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pbOpenFolder">
     <property name="geometry">
      <rect>
       <x>882</x>
       <y>24</y>
       <width>39</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>📂</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>28</y>
       <width>97</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Export folder</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="bStartProcess">
    <property name="geometry">
     <rect>
      <x>418</x>
      <y>416</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>444</y>
      <width>937</width>
      <height>19</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>943</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
